src/@types/inews/index.d.ts

```diff
@@ -28,6 +28,7 @@ declare module 'inews' {
 			cumeTime?: string // number
 			backTime?: string // @number (seconds since midnight)
 			layout?: string
+			skema?: string
```

Comment:
I might be wrong, but i thought we wanted to use the english word in code and say `schema?: string` instead?

Reply:
We do want to use english words. However, this is the attribute sent from iNews so if we don't name it `skema` we wont be able to retrieve the data.

Everywhere else I should have named it `schema` (if not - then don't hesitate to point it out)